analyzers/tests/SonarAnalyzer.UnitTest/TestCases/GenericReadonlyFieldPropertyAssignment.CSharp10.cs

```diff
@@ -15,6 +15,15 @@ public PointManager(T point)
 
     public void MovePointVertically((int, int) p)
     {
-        (point.X, int y) = p; // Compliant - FN: if point is a struct, then nothing happened
+        (point.X, int y) = p; // Noncompliant {{Restrict 'point' to be a reference type or remove this assignment of 'X'; it is useless if 'point' is a value type.}}
```

Comment:
Could you please add a codefix provider test for CSharp10, to verify if the whole assignment is removed correctly?

Reply:
Not the best of options and a bit hacky but I suggest to do the following:

- Open an issue which documents the missing functionality from the testing infrastructure
- Create a test that documents that after running the codefix provider on the .CSharp10 test case it is not changed. This should be enough to document that the codefix provider did not offer a fix for the issue.
- Open an issue on github for the missing functionality for the codefix provider, and we will tackle it in the future.

Reply:
I create #5768 to track it